src/runtime_src/core/pcie/tools/xbmgmt/cmd_flash.cpp

```diff
@@ -254,17 +254,25 @@ static int resetShell(unsigned index, bool force)
     return flasher.upgradeFirmware("", nullptr, nullptr, nullptr);
 }
 
-/* We do not take the risk to flash any bmc marked as UNKNOWN */
+/*
+ * bmcVer (shown as [SC=version]) can be 3 status:
+ *   1) regular SC version;
+ *        example: [SC=4.1.7]
+ *   2) INACTIVE;
+ *        exmaple: [SC=INACTIVE], this means no xmc subdev, we should not
+ *        attemp to flash the SC;
+ *   3) UNKNOWN;
+ *        example: [SC=UNKNOWN], this means xmc subdev is online, but status in
+ *        not normal, we still allow flashing SC.
+ */
```

Comment:
do we want to document the fourth case: fixed?

Reply:
Yes, we do.

I will use another PR to adjust it. Not sure when it will be ready, SC team is still working on releasing another version; and hotreset on U30 should be fixed too.

Reply:
ok, sounds good.